pkg/util/uuid/uuid.go

```diff
@@ -0,0 +1,27 @@
+package uuid
+
```

Comment:
You are missing Microsoft license here. Run all tests locally for it to be generated or just add it from other files.